models/marts/core/dim_customers.sql

```diff
@@ -0,0 +1,40 @@
+with customers as (
+    select * from {{ ref('stg_customers')}}
+),
+orders as (
+    select * from {{ ref('stg_orders')}}
+),
+customer_orders as (
+    select
+        customer_id,
+        min(order_date) as first_order_date,
+        max(order_date) as most_recent_order_date,
+        count(order_id) as number_of_orders,
+        sum(amount) as lifetime_value_total
+    from analytics.analytics.orders
+    group by 1
+),
+lifetime_value as (
+    select 
+        customer_id,
+        sum(amount) as lifetime_value_balance 
+    from analytics.analytics.orders

```

Comment:
>   from analytics.analytics.orders

Did you actually want the customers model here?

Reply:
now that I understand that more I think payments model perhaps